xblockgrademe/private/view.js

```diff
@@ -4,8 +4,32 @@ function XblockGrademeView(runtime, element) {
     var $ = window.jQuery;
     var $element = $(element);
 
-    // TODO: Put your logic here
+    var failure_message = "We are sorry; users who have not created accounts and registered " +
+                          "for the course may not receive a Statement of Accomplishment.";
+    
+    var success_message = "Your grading request has been received. If you have passed, your Statement of " +
+                          "Accomplishment should be available on your <a href='/dashboard'>account dashboard</a> " +
+                          "page within the next 20 minutes.";
```

Comment:
I prefer to use doubles for long strings that might at some point need to contain apostrophes...